AppCenterReactNativeShared/android/src/main/java/com/microsoft/appcenter/reactnative/shared/AppCenterReactNativeShared.java

```diff
@@ -26,13 +28,16 @@ public static void configureAppCenter(Application application) {
         if (AppCenter.isConfigured()) {
```

Comment:
What we can do instead is check for sApplication != null here instead of checking isConfigured, to avoid resetting static fields.